src/Cake.NuGet/NuGetContentResolver.cs

```diff
@@ -72,6 +76,11 @@ private IReadOnlyCollection<IFile> GetAddinAssemblies(DirectoryPath path, Packag
                 // Get relative path.
                 var relative = path.GetRelativePath(assembly.Path);
                 var framework = ParseFromDirectoryPath(current, relative.GetDirectory());
+                if (framework == null)
+                {
+                    _log.Warning("Unable to parse compatible framework from path {0}. Setting any framework.", assembly.Path.MakeAbsolute(_environment).FullPath);
```

Comment:
@mholo65 I'm not @gep13 (obviously), but I think that `Setting any framework.` sounds a little bit "leaky" in the sense that we leak internal information to the consumer which probably won't understand what we're talking about.

Perhaps we could change it to something similar to `Falling back to using root folder of NuGet package.` or something like that.

Thoughts @gep13?